lib/scsi_defs.h

```diff
@@ -156,7 +156,7 @@ static_assert(sizeof(ReadCapacity10Data) == 8);
 struct InquiryCommand {
   uint8_t reserved : 6;
   bool obsolete : 1;  // formerly CMDDT
-  bool evpd : 1;      // Enable Vital Product Data (EVPD)
+  bool evpd : 1;      // Enable Vital Product Data (EVpd)
```

Comment:
Is 'EVpd' a standard abbreviation for 'Enable Vital Product Data'? Or is it internal to our project?

Reply:
Its a standard abbreviation it seems like as I found this on https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf section 3.6.1

Reply:
I just followed what was on the SCSI doc. That un-capitalization was accidental though.

Reply:
I was referring to the change in capitalization. Thanks